src/test/java/fi/metatavu/kuntaapi/test/server/integration/ptv/TestPtvConsts.java

```diff
@@ -2,76 +2,351 @@
+    "b8931c4b-1119-4c69-9bc4-a41fb00b92fe",
+    "7de53057-287b-4899-b4b3-86291bf47b45"
   };
 
-  public static final String[] ELECTRONIC_CHANNEL_SERVICE_CHANNELS = {
-    "b8e980f4-58c7-477b-a9d1-b01d50be50f0",
-    "614bdc8a-bb55-4f36-be4e-7e1bea9a7706"
+  public final static String[] ELECTRONIC_CHANNEL_SERVICE_CHANNELS = {
+    "6631f997-eb58-401e-b0a9-274f7d51cb0a",
+    "2f648147-a615-4d31-b251-f4fbe3a26ee4",
+    "1d01e828-362b-4cd2-b302-c3a4b8b1941c",
+    "7374dd89-8e26-4402-90fe-c32942525476",
+    "daee70a8-ec2d-413d-ab79-2ebf79fe04a3",
+    "3e0951ab-890f-441b-bcf5-5c27b56b1fde",
+    "80f6bcb5-e224-4041-97da-95b6936878fe",
+    "a165743c-4121-4ab5-b1d4-e04ff36ee430",
+    "b8931c4b-1119-4c69-9bc4-a41fb00b92fe"
   };
 
-  public static final String[] SERVICE_LOCATION_SERVICE_CHANNELS = {
-    "0452df6a-f322-47de-af5b-767694f120cd",
-    "faa9c7b7-5b1f-4964-b04f-410558b66647",
-    "97c01787-b57c-42f1-a002-7d09b363993f",
-    "b5fba182-2a08-44aa-a168-c11f8d1d4664",
-    "dda27a67-1b20-4966-a57e-162ca4c54ef5",
-    "4d52960e-2e39-4d9e-8f7c-5bc8994b024f",
-    "1dec7c89-96b8-4860-83d0-bad8b8897ba6",
-    "a31fbcc7-d45f-4eb2-a32a-afd134d02aa5",
-    "155b896e-f1f5-43db-9497-aa3a8afd2487",
-    "0e4abaca-f57e-4649-a42d-0f421d09e9fd"
+  public final static String[] SERVICE_LOCATION_SERVICE_CHANNELS = {
+    "1bd39cda-3006-482e-acff-93ab8ddc9f1b",
+    "13992280-9c95-4ede-af13-33ad2acde746",
+    "32833c27-7d41-4267-800a-a9a888b513b7",
+    "2481937a-5d4d-4643-a9b9-9877be5d4d3e",
+    "3cfd7097-b73e-4f2a-a482-099b00f12b02",
+    "5d04a7cd-d372-409e-b3d6-211ae5325bd2",
+    "69c2f81a-bc28-4992-9491-b82dacc0868c",
+    "ecdf1bc2-ad32-46cc-9421-50ce61bc710f",
+    "41ebe689-dbed-4bb2-a6f2-184e29d625ed",
+    "abac25e1-bd15-4c1a-b22e-9d9a705d9220"
   };
 
-  public static final String[] PRINTABLE_FORM_SERVICE_CHANNELS = {
-    "047b91ef-aa08-4be6-b72d-1231e61c5cb9",
-    "1af0ec8c-b647-4f40-b1a9-db6052c5096d"
+  public final static String[] PRINTABLE_FORM_SERVICE_CHANNELS = {
+    "c38feebb-9dfa-4efc-b2f7-02b2fd282841",
+    "14a9874a-8b9b-488a-aec5-5321dd36678f",
+    "7de53057-287b-4899-b4b3-86291bf47b45"
   };
 
-  public static final String[] PHONE_SERVICE_CHANNELS = {
-    "0c00262c-3529-4cc3-b0fe-2f03a2c4e4ce",
-    "b0ca2d94-f113-47ae-9c90-1bc1733c44be"
+  public final static String[] PHONE_SERVICE_CHANNELS = {
+    "bc91db9e-2f56-4044-a375-30987de714eb",
+    "2e7f0b8d-4943-4fd4-8a22-7b02830be16e",
+    "a4ba85e1-0b1f-4f40-a7fd-0ed7342e0547",
+    "6dc68be3-cfc9-44b5-afc4-adc9fae54763",
+    "4faaad5e-5cba-4166-a76b-a2f0a0ba64eb",
+    "54cea689-0d6e-41ac-adf6-dd12e7e97769",
+    "ad29e11d-9c97-4e02-a7da-5f5cefe442e9",
+    "403097f7-f397-4eed-aac6-98442faa33b6",
+    "2877aead-1169-407b-8123-a3bbe5cfe458",
+    "6894ee49-f5ac-4a69-b9f0-bf3de2a02f0c"
   };
 
-  public static final String[] WEB_PAGE_SERVICE_CHANNELS = {
-    "74b3ac81-cf57-40b0-b0b1-b9dcf752a143"
+  public final static String[] WEB_PAGE_SERVICE_CHANNELS = {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AModifiersOrderCheck)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this member "protected". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2386)